languagetool-language-modules/es/src/main/resources/org/languagetool/rules/es/remote-rule-filters.xml

```diff
@@ -809,14 +804,14 @@ To ignore a remote rule match, set the <marker> so that it exactly covers the te
                 <example correction="">No quiero más, ya que <marker>sí</marker> afecta a mi salud.</example>
                 <example correction="">Aunque librosramona.com <marker>sí</marker> guarda un registro de las solicitudes.</example>
                 <example correction="">Me siento frente a la pantalla de un computador el día entero, así que <marker>sí</marker> resulto enormemente bombardeado por ondas electromagnéticas.</example>
-                <example>Dijo que si puedo detener una Señal, que la cadena se romperia</example>
-                <example>Le digo a La Heredera que si podemos jugar juntas.</example>
-                <example>de modo que si observamos alguna incomodidad o alguna reacción exagerada, poder parar para atender sus necesidades</example>
-                <example>Prestaban más atención que si gritaba sin parar. </example>
-                <example>Es más agradable a la vista que si lo tachas concienzudamente.</example>
-                <example>Son menos que las posibles opciones si aplicamos las 27 letras del alfabeto.</example>
-                <example>No es igual que si tienen algo que ocultar.</example>
-                <example>Vaya que si ha sido dramático.</example>
+                <example>Dijo que sí puedo detener una Señal, que la cadena se romperia</example>
+                <example>Le digo a La Heredera que sí podemos jugar juntas.</example>
+                <example>de modo que sí observamos alguna incomodidad o alguna reacción exagerada, poder parar para atender sus necesidades</example>
+                <example>Prestaban más atención que sí gritaba sin parar. </example>
+                <example>Es más agradable a la vista que sí lo tachas concienzudamente.</example>
+                <example>Son menos que las posibles opciones sí aplicamos las 27 letras del alfabeto.</example>
+                <example>No es igual que sí tienen algo que ocultar.</example>
+                <example>Vaya que sí ha sido dramático.</example>
             </rule>
```

Comment:
The same. If the model tries to change sí -> si in these sentences, the rule in remote-rule-filter.xml will not prevent the model from doing it.